lib/ansible/plugins/test/success.yml

```diff
@@ -2,11 +2,11 @@ DOCUMENTATION:
   name: success
   author: Ansible Core
   version_added: "1.9"
-  short_description: check task success
+  short_description: check task success or skipped
   aliases: [succeeded, successful]
   description:
-    - Tests if task finished successfully, opposite of C(failed).
```

Comment:
I think this line should probably stay, we can clarify like you've done later on that a skipped result is also considered a success.

Reply:
This is covered in the other thread: https://github.com/ansible/ansible/pull/83705#discussion_r1706136358